src/main/kotlin/com/terraformation/backend/tracking/api/PlantingSitesController.kt

```diff
@@ -137,6 +171,25 @@ data class CreatePlantingSiteResponsePayload(val id: PlantingSiteId) : SuccessRe
 
 data class GetPlantingSiteResponsePayload(val site: PlantingSitePayload) : SuccessResponsePayload
 
+data class ListPlantingSiteSpeciesResponsePayload(
+    @ArraySchema(
+        arraySchema =
+            Schema(
+                description =
+                    "List of species for each planting zone. A species may appear in more than " +
+                        "one planting zone."))
+    val plantingSubzones: List<PlantingSubzoneSpeciesElement>,
+) : SuccessResponsePayload {
+  constructor(
+      speciesBySubzone: Map<PlantingSubzoneId, List<ExistingSpeciesModel>>
```

Comment:
Should this be 'Zone' instead of 'Subzone', that being the layer immediately within planting sites (and also congruent with the description)?

Reply:
Or maybe the description should say "List of species for each planting zone"?